theia-electron/package.json

```diff
@@ -0,0 +1,64 @@
+{
+    "private": true,
+    "name": "theia-electron",
+    "version": "0.3.16",
+    "description": "Theia",
+    "main": "src-gen/frontend/electron-main.js",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/theia-ide/theia.git"
+    },
+    "theia": {
+        "target": "electron"
+    },
+    "scripts": {
+        "prepare": "yarn run build",
+        "build": "theia build",
+        "package": "electron-builder",
+        "package:win": "electron-builder -w",
+        "package:mac": "electron-builder -m",
+        "package:linux": "electron-builder -l",
+        "package:all": "electron-builder -lmw",
+        "package:preview": "electron-builder --dir"
+    },
+    "keywords": [],
+    "author": "typefox <npm@typefox.io>",
+    "license": "Apache-2.0",
+    "bugs": {
+        "url": "https://github.com/theia-ide/theia/issues"
+    },
+    "homepage": "https://github.com/theia-ide/theia#readme",
+    "dependencies": {
+        "@theia/callhierarchy": "next",
+        "@theia/core": "next",
+        "@theia/editor": "next",
+        "@theia/file-search": "next",
+        "@theia/filesystem": "next",
+        "@theia/git": "next",
+        "@theia/java": "next",
+        "@theia/json": "next",
+        "@theia/keymaps": "next",
+        "@theia/languages": "next",
+        "@theia/markers": "next",
+        "@theia/merge-conflicts": "next",
+        "@theia/messages": "next",
+        "@theia/mini-browser": "next",
+        "@theia/monaco": "next",
+        "@theia/navigator": "next",
+        "@theia/outline-view": "next",
+        "@theia/output": "next",
+        "@theia/preferences": "next",
+        "@theia/preview": "next",
+        "@theia/process": "next",
+        "@theia/search-in-workspace": "next",
+        "@theia/task": "next",
+        "@theia/terminal": "next",
+        "@theia/typescript": "next",
```

Comment:
@kittaakos we expect that typescript is already installed on the machine? Can we bundle one version as a fallback?

Reply:
I am not against bundling it, but it is not necessary. https://github.com/theia-ide/theia/issues/3568

Reply:
Let's do it. I do not mind.